src/utils/formatPriceImpact.test.ts

```diff
@@ -0,0 +1,11 @@
+import { Percent } from "@uniswap/sdk-core"
+import formatPriceImpact from "./formatPriceImpact"
+
+describe('formatPriceImpact', () => {
+  it('formats positive price impact', () => {
+    expect(formatPriceImpact(new Percent(5, 10_000))).toEqual('-0.05%')
+  })
```

Comment:
thereotically no such thing as positive price impact (sometimes this does show up cuz bug in routing-api lol) so i think you can just take out this test case

Reply:
if it _can_ occur I still think it's good to test for it.